README.md

```diff
@@ -113,12 +113,9 @@ npm start
 # Build
 npm run docker:build
 
-# Run on Linux or macOS
+# Run
 npm run docker:run
 
-# Run on Windows (you can replace "UTC" by your time zone)
-docker run -e TZ=UTC -p 1337:1337 -it leonai/leon
-
```

Comment:
Did you try to run it with a Windows OS to see if it breaks without the `TZ` environment variable?

Reply:
To be honest, no.
Thanks for this catch, I'll test if it's work, if not, I would like that it stays straight forward, doesn't matter the OS.

Yeah, I'll look at it, and will try to understand the issue. :+1:

Reply:
Awesome! Looking forward to see your feedback :+1: